### ISSUE TITLE
[Feature Request] Announce when maxlength is reached

### ISSUE THREAD

**Issue by rkrasnan** [vuetifyjs/vuetify#10017]
### Problem to solve
A non-sighted user is not informed when they have reached the maxlength in a v-textarea or v-text-field and may continue typing, believing they are still entering information. 

### Proposed solution
Add a visually hidden aria-live section to the v-text-field and v-textarea components that updates when the maxlength is reached saying "You have reached the maximum input for this field"

**Comment by KaelWD** [comment]
Related: https://github.com/nvaccess/nvda/issues/7910

Are there any examples of this implemented in other component libraries?

**Comment by rkrasnan** [comment]
I don't think other component libraries do this. 
It does seem like it should be handled by the screen reader but right now it isn't.

**Comment by phiter** [comment]
I don't think we need to implement this. It's a screenreader issue as @KaelWD linked.

**Comment by johnleider** [comment]
This Issue is being closed due to inactivity.

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
